datalake/backend/tests/test_import_dataset_logging.py

```diff
@@ -0,0 +1,126 @@
+import json
+import logging
+from json import dumps
+from unittest.mock import MagicMock, patch
+
+from jsonschema import ValidationError  # type: ignore[import]
+from pytest import mark
+from pytest_subtests import SubTests  # type: ignore[import]
+
+from ..processing.import_dataset.task import lambda_handler
+from .utils import (
+    ProcessingAsset,
+    any_dataset_id,
+    any_dataset_version_id,
+    any_hex_multihash,
+    any_lambda_context,
+    any_s3_url,
+)
+
+
+class TestLogging:
+    logger: logging.Logger
+
+    @classmethod
+    def setup_class(cls) -> None:
+        cls.logger = logging.getLogger("datalake.backend.processing.import_dataset.task")
+
+    @mark.infrastructure
+    @patch("datalake.backend.processing.import_dataset.task.s3control_client.create_job")
+    def test_should_log_payload(self, create_job_mock: MagicMock) -> None:
```

Comment:
I think if you `patch("datalake.backend.processing.import_dataset.task.validate")` (probably at line 41, not here, since we shouldn't need the validate mock for anything) this doesn't need to be `@mark.infrastructure`, meaning it'll run much faster and can run locally without deploying.